docker/data-science/notebooks/ch07/1.index-datasets.ipynb

```diff
@@ -284,16 +291,9 @@
      "text": [
       "Wiping \"entities\" collection\n",
       "Creating \"entities\" collection\n",
-      "http://aips-solr:8983/solr/entities/schema\n",
-      "{\n",
-      "  \"responseHeader\":{\n",
-      "    \"status\":0,\n",
-      "    \"QTime\":143\n",
-      "  }\n",
-      "}\n",
       "Status: Success\n",
-      "Loading entities\n",
-      "entities Schema: \n",
+      "Loading file\n",
```

Comment:
Why is it sometimes "file" (i.e. "entities" is now gone) and sometimes the name is there (i.e. Geonames below). Should we always return the filename instead of just "file"?